stripe/src/main/java/com/stripe/android/networking/RetryDelaySupplier.kt

```diff
@@ -0,0 +1,29 @@
+package com.stripe.android.networking
+
+import java.util.concurrent.TimeUnit
+import kotlin.math.pow
+
+internal class RetryDelaySupplier(
+    private val incrementSeconds: Long = DEFAULT_INCREMENT_SECONDS
+) {
+    /**
+     * Calculate an exponential backoff delay before retrying the next completion request.
+     *
+     * Delay 2 seconds before the first retry
+     * Delay 4 seconds before the second retry
+     * Delay 8 seconds before the third retry
```

Comment:
since this class no longer knows max retry is 3, please update this part to the actual formula